packages/ruby/spec/readme/har_spec.rb

```diff
@@ -104,13 +105,13 @@
         Rack::Response.new(response_body, status_code, headers),
         Time.now,
         Time.now + 1,
-        ["Filtered-Header", "key1"]
+        Filter.for(["Filtered-Header", "key1"])
       )
 
       json = JSON.parse(har.to_json)
 
       response = json.dig("log", "entries", 0, "response")
-      response_body = response["content"]["text"]
+      response_body = JSON.parse(response["content"]["text"])
```

Comment:
Again, I think this is saying that the body is doubly encoded, which we don't want?